versions.tf

```diff
@@ -0,0 +1,4 @@
+
+terraform {
```

Comment:
Is this block not already defined inside `main.tf`?

Reply:
The terraform 0.12upgrade command creates this file when it upgrades stuff. At the time of the comment, both existed. I've since moved everything here, since apparently that's what hashicorp wants to be the norm (though I'm sure it doesn't matter either way).